services/data/src/links/RestAPILink/fetchData.test.ts

```diff
@@ -109,7 +109,11 @@ describe('networkFetch', () => {
     describe('fetchData', () => {
         const headers: Record<string, (type: string) => string> = {
             'Content-Type': type =>
-                type === 'json' ? 'application/json' : 'text/plain',
+                type === 'json'
+                    ? 'application/json'
+                    : type === 'text'
+                    ? 'text/plain'
+                    : 'image/png',
```

Comment:
Hmmm... should `image/png` be the default here?  Seems like that should be a quite specific case, and either `text/plain` (or nothing) should be the default...

Reply:
Yeah possibly a good point. TBH, I found this test setup a little bit quirky and just tried to add to it. Looking at the function we are testing, I'd say this order of cases makes sense, because in `fetchData` we:
1. Return `response.json()` if the content type is `application/json`
2. Return `response.text()` if the content type is `text/*`
3. By default we return `response.blob()`

In my mind, all that is important for testing that default case is that we end up with a content-type that does not fall under 1 or 2.

Perhaps it would have been clearer if I had just made up a content-type string so it doesn't create any false expectations?

Reply:
For example:
```js
        const headers: Record<string, (type: string) => string> = {
            'Content-Type': type =>
                type === 'json'
                    ? 'application/json'
                    : type === 'text'
                    ? 'text/plain'
                    : 'some/other-content-type',
        }
```

Reply:
Ah yeah I kind of missed that this was in a test (my bad) - either way seems fine to me, then